msgid ""
msgstr ""
"Project-Id-Version: pylav\n"
"POT-Creation-Date: 2024-05-07 00:16+0000\n"
"PO-Revision-Date: 2024-05-23 10:22\n"
"Last-Translator: \n"
"Language-Team: Lithuanian\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: redgettext 3.4.2\n"
"Plural-Forms: nplurals=4; plural=(n%10==1 && (n%100>19 || n%100<11) ? 0 : (n%10>=2 && n%10<=9) && (n%100>19 || n%100<11) ? 1 : n%1!=0 ? 2: 3);\n"
"X-Crowdin-Project: pylav\n"
"X-Crowdin-Project-ID: 535244\n"
"X-Crowdin-Language: lt\n"
"X-Crowdin-File: /lib/pylav/extension/bundled_node/locales/messages.pot\n"
"X-Crowdin-File-ID: 3385\n"
"Language: lt_LT\n"

#: pylav/extension/bundled_node/manager.py:239
msgid "You are attempting to run the managed Lavalink node on an unsupported machine architecture."
msgstr "Jūs bandote paleisti valdomą Lavalink mazgą nepalaikomoje mašinos architektūroje."

#: pylav/extension/bundled_node/manager.py:466
msgid "The output of `{command_name_variable_do_not_translate}` was unexpected\n"
"{command_output_variable_do_not_translate}"
msgstr "\" produkcija{command_name_variable_do_not_translate}\" buvo netikėta\n"
"{command_output_variable_do_not_translate}"

#: pylav/extension/bundled_node/manager.py:489
msgid "The port {port_variable_do_not_translate} is already in use. Managed Lavalink startup has been aborted."
msgstr "Uostas {port_variable_do_not_translate} jau naudojamas. Valdomas \"Lavalink\" startuolis buvo nutrauktas."

#: pylav/extension/bundled_node/manager.py:494
msgid "Lavalink failed to start: {error_variable_do_not_translate}"
msgstr "\"Lavalink\" nepavyko paleisti: {error_variable_do_not_translate}"

